ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dhcp-relay.j2

```diff
@@ -0,0 +1,25 @@
+{% if dhcp_relay is arista.avd.defined %}
+
+# DHCP Relay
+
+## DHCP Relay Summary
+{%     if dhcp_relay.tunnel_requests_disabled is arista.avd.defined(true) %}
+
+- DHCP Relay is disabled for tunnelled requests
+{%     elif dhcp_relay.tunnel_requests_disabled is arista.avd.defined(false) %}
+
+- DHCP Relay is enabled for tunnelled requests
+{%     endif %}
+
+| Server |
```

Comment:
Can you add a defined test for ```dhcp_relay.servers``` above this line to avoid printing the ```| Server |``` if servers list is not defined.

Reply:
Fixed.